ra2ce/analysis/analysis_config_data/analysis_config_data_reader.py

```diff
@@ -145,11 +145,6 @@ def _get_analysis_section_losses(self, section_name: str) -> AnalysisSectionLoss
             "duration_event",
             fallback=_section.duration_event,
         )
```

Comment:
```suggestion
        )
        _section.hours_per_day = self._parser.getfloat(
        section_name,
        "hours_per_day",
        fallback=_section.hours_per_day,
        )
```

I suggest to keep hours_per_day but not use it. Please see remark 1.